tests/Integration/Migration/fixtures/2014_10_12_000000_create_members_table.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace Illuminate\Tests\Integration\Migration\fixtures;
```

Comment:
This is unnecessary as PHPUnit doesn't need composer autoload for this to work, The migrator would just `glob()` on the related path and require it manually.

Reply:
However I do understand that the issue was there already a duplicate migration class under <https://github.com/laravel/framework/blob/master/tests/Integration/Database/stubs/2014_10_12_000000_create_members_table.php>.

May I suggest running `$this->loadLaravelMigrations(['--database' => 'testbench']);` instead.

This would simply load the default Laravel migration found under <https://github.com/orchestral/testbench-core/blob/master/laravel/migrations/>

Reply:
`$this->loadLaravelMigrations` doesn't just load the migrations, but also run them via Artisan, which is precisely the thing I'm avoiding in this test.
I want to prove that the Migrator class works outside of Artisan.